test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,100 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 name=<aws-iam-user>"
```

Comment:
When I tested this, I got a warning:

```console
[WARNING]: Found variable using reserved name: name
```

Reply:
Fixed - replaced with the variable `vpc_name` instead.